manifests/07-downloads-deployment.yaml

```diff
@@ -89,10 +89,14 @@ spec:
               os.mkdir(arch)
               for operating_system in ['linux', 'mac', 'windows']:
                   os.mkdir(os.path.join(arch, operating_system))
+                  target_path = os.path.join(arch, operating_system, 'LICENSE')
+                  os.symlink('/usr/share/openshift/LICENSE', target_path)
```

Comment:
Do we need to have a separate `LICENSE` file for each os? Can't hurt but II guess that they will be all the same.

Reply:
It's the same source for all, so the same license. In fact you could just use a local copy of the Apache 2.0 license and call it a day :) I don't see the license changing ever.